test/converters/html-converter.js

```diff
@@ -22,23 +22,23 @@ describe('HtmlConverter', function () {
   describe('# Functionality', function () {
     it('should be able to convert the given JSON to HTML using the given template and engine', async () => {
       const templateLoader = new TemplateLoader();
-      const converter = new HtmlConverter(templateLoader);
+      const converter = new HtmlConverter({templateLoader: templateLoader});
```

Comment:
This can actually be written shorter:
```
const converter = new HtmlConverter({templateLoader: templateLoader});
```
is the same as 
```
const converter = new HtmlConverter({templateLoader});
```